packages/amplify-category-api/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.js

```diff
@@ -634,7 +635,8 @@ function newLambdaFunction(context, path) {
       },
     },
   };
-  return add(context, 'awscloudformation', 'LambdaFunction', params).then(resourceName => {
+
+  return add(context, 'awscloudformation', ServiceName.FunctionServiceName, params).then(resourceName => {
```

Comment:
should this be FunctionServiceName.LambdaFunction?

Reply:
Yes! Not sure how I made that mistake... fixed